integration_tests/pages/page.ts

```diff
@@ -4,8 +4,8 @@ import logAccessibilityViolations from '../support/logAccessibilityViolations'
 export type PageElement = Cypress.Chainable<JQuery>
 
 export default abstract class Page {
-  static verifyOnPage<T>(constructor: new () => T): T {
-    return new constructor()
+  static verifyOnPage<T>(constructor: new (title?: string) => T, title?: string): T {
```

Comment:
Updating this so we can optionally pass a title for integration test Page object constructor, for cases where page titles vary.